### DISCUSSION TITLE
No IntelliSense for constructor of std::unordered_set.

### DISCUSSION THREAD

**Discussion by tasty0tomato** [microsoft/vscode-cpptools#10106] [Show and tell]
<img width="345" alt="Code_n35LkuRrU2" src="https://user-images.githubusercontent.com/48764449/199689902-c6954a30-1263-425f-ac59-3d2377b87504.png">
Want to have .begin() and .end()

**Comment by sean-mcmanus** [comment]
It works with `std::unordered_set{wordDict.}`. I'm not sure yet if that's "by design" or not.

**Reply by sean-mcmanus** [reply]
The fix is available with 1.14.0 (pre-release): https://github.com/microsoft/vscode-cpptools/releases/tag/v1.14.0
